plugins/action_revoke_permissions/src/revokePermissions.ts

```diff
@@ -6,14 +6,16 @@ const revokePermissions = async ({ applicationData, parameters, DBConnect }: Act
   const db = databaseMethods(DBConnect)
   // Don't specify orgName/Id default because we might be targeting a permission_join with no user (even if application has organisation)
   const { username = applicationData?.username, orgName, orgId, permissionNames } = parameters
+  const isRemovingPermission = true
```

Comment:
```suggestion
  const { username = applicationData?.username, orgName, orgId, permissionNames, isRemovingPermission = true } = parameters
```
If you move this value into parameters (but with default), it'll do exactly the same thing but allows us to do it the other way if/when we need to without updating this again.